### PR TITLE
Memory watch

### LINKED ISSUE #10: Memory watch
- Provide a tool to see the content of variables directly on screen.

### PR DESCRIPTION
**Pull request by GTcreyon**

### CODE REVIEW

**Review by KevinKib — COMMENTED** [review]
Also it'd be better if this branch could be merged into `dev` and not into `display` since those branches aren't linked together, but I guess I'll see that when I merge

File: src/classes/TextManager.as [hunk 1]
```diff
@@ -1,53 +1,35 @@
...
     public function TextManager() {
-        this.row1 = "";
-        this.row2 = "";
-        this.row3 = "";
-        this.row4 = "";
-        this.row5 = "";
-        this.row6 = "";
+        var i = 0;
+        for(i = 0; i < 6; i++)
+        {
```
Comment (KevinKib) [inline src/classes/TextManager.as:13]: How many rows are available now? I guess that number can be changed?

File: docs/Commands.md [hunk 2]
```diff
@@ -460,6 +460,30 @@ Writing 'lives infinite' will prevent Mario from ever game-overing.
...
+**Syntax :** addwatchvar / awv [name]
+
+* **name**: Name of the variable to watch for.
+
+***
+</details>
+
+<details>
+<summary>removewatchvar</summary>
+
```
Comment (KevinKib) [inline docs/Commands.md:480]: similarly; `removewatch`?

File: docs/Commands.md [hunk 3]
```diff
@@ -460,6 +460,30 @@ Writing 'lives infinite' will prevent Mario from ever game-overing.
 ***
 </details>
 
+<details>
+<summary>addwatchvar</summary>
+
+> Adds a variable to the memory watch.
+
+**Syntax :** addwatchvar / awv [name]
+
```
Comment (KevinKib) [inline docs/Commands.md:470]: Maybe `watchvar` or `watch` instead? I feel like both of these convey the idea pretty well (and we are only watching variables anyways)

**Review by KevinKib — CHANGES_REQUESTED** [review]

File: src/classes/TextManager.as (hunk 1 above)
Comment (GTcreyon) [reply src/classes/TextManager.as:13]: As many rows as required, technically, but the memory watch just uses '\n' at the end of each entry.

Comment (KevinKib) [reply src/classes/TextManager.as:13]: Okay, we'll let this system until another command wants one of these rows (if that ever happens).

**Review by KevinKib — APPROVED** [review]

**Pull request merged**